src/app/static/src/app/components/regionPage.vue

```diff
@@ -23,8 +23,9 @@
     import {RootMutation} from "../mutations";
     import {mapMutationByName} from "../utils";
     import stepButton from "./stepButton.vue";
-    import interventions from "./interventions.vue";
     import baseline from "./baseline.vue";
+    // @ts-ignore
```

Comment:
The typescript docs suggest listing what is being ignored a the end of the comment (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html)